public/app/plugins/datasource/elasticsearch/configuration/DataLinks.tsx

```diff
@@ -29,60 +33,63 @@ export const DataLinks = (props: Props) => {
   const styles = useStyles2(getStyles);
 
   return (
-    <>
-      <h3 className="page-heading">Data links</h3>
+    <ConfigSubSection
+      title="Logs"
```

Comment:
this this supposed to be "Data links" instead of logs?

Reply:
Yes! Good catch 🤝